src/commons/error.rs

```diff
@@ -111,6 +111,45 @@ impl fmt::Display for RoaDeltaError {
     }
 }
 
+// ApiAuthError is *also* implemented as a separate enum,
+// so that we don't have to implement the Clone trait for
+// all of the Error enum.
+// Also it makes kind of sense to keep these errors as a separate container,
+// since they all originate in interactions with the OIDC provider (or lack thereof)
+#[derive(Debug, Clone)]
+pub enum ApiAuthError {
+    ApiInvalidCredentials(String),
+    ApiLoginError(String),
```

Comment:
I don't have a better suggestion but some of the other errors could be considered to be "login errors" so I'm thinking we should revisit this name.

Reply:
I wanted to differentiate between errors _only_ occurring at login and errors that could occur _both_ at login and at (re)authentication time.